### DISCUSSION TITLE
Wiki.js SSO - DNS log in issues

### DISCUSSION THREAD

**Discussion by Frixs** [requarks/wiki#7020] [Error / Bug Report]
Hello, I made the post about the issue already on Reddit here:
https://www.reddit.com/r/selfhosted/comments/1axkxh1/synology_sso_wikijs_dns_issues/
_You can find all the images and descriptions in there._

Basically, when I set up my SSO Client and SSO Synology Server, it works perfectly when I use IP addresses.
However, when I want to use my local DNS, it does not work. **The SSO Server logs me in, but when I get redirected back, it fails to log me on the WIki.js side.**

I tried to look into the WIki.js authentication code, but nothing that I could change. Auth logic is hidden within passport-oidc lib, if I'm not mistaken.

Any ideas or help? This seems like a bug.

---

What I would like to achieve is that I have my NAS on my local network as 192.168.0.99 and then I use Tailscale where my NAS has different IP. I want to be able to use SSO on both networks.

**Comment by NGPixel** [comment]
Did you change the wiki host under admin > general?
